src/site/content/en/blog/houdini-how/index.md

```diff
@@ -0,0 +1,147 @@
+  <img class="w-screenshot" src="caniuse-paint.png" alt="Caniuse support">
+  <figcaption class="w-figcaption">
+    The CSS Painting API is currently supported on Chromium-based browsers.</a>
+  </figcaption>
+</figure>
+
+But even without full browser support, you can still get creative with the Houdini Paint API and see your styles work across all modern browsers with the [CSS Paint Polyfill](https://github.com/GoogleChromeLabs/css-paint-polyfill). And to showcase a few unique implementations, as well as to provide a resource and worklet library, my team built [houdini.how](https://houdini.how).
+
+## Houdini.how
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="worklet-page.png" alt="Worklet page screenshot.">
+  <figcaption class="w-figcaption">
+    Screenshot from the <a href="https://houdini.how">Houdini.how</a> homepage.
+  </figcaption>
+</figure>
+
+[Houdini.how](https://houdini.how) is a library and reference for Houdini worklets and resources. It provides everything you need to know about CSS Houdini: browser support, an [overview](https://houdini.how/about) of its various APIs, [usage](https://houdini.how/usage) information, additional [resources](https://houdini.how/resources), and live paint worklet [samples](https://houdini.how/). Each sample on Houdini.how is backed by the CSS Paint API, meaning they each work on all modern browsers.
+Give it a whirl!
+
+## Using Houdini
+
+Houdini worklets must either be run via a server locally, or on HTTPS in production. In order to work with a Houdini worklet, you will need to either install it locally or use a content delivery network (CDN) like [unpkg](https://unpkg.com) to serve the files. You will then need to register the worklet locally.
+
+There are a few ways to include the Houdini.how showcase worklets in your own web projects. They can either be used via a CDN in a prototyping capacity, or you can manage the worklets on your own using npm modules. Either way, you'll want to also include the CSS Paint Polyfill to ensure they are cross-browser compatible.
+
+### 1. Prototyping with a CDN
+
+When registering from unpkg, you can link directly to the worklet.js file without needing to locally install the worklet. Unpkg will resolve to the worklet.js as the main script, or you can specify it yourself. Unpkg will not cause CORS issues, as it is served over HTTPS.
+
+```html
+CSS.paintWorklet.addModule("https://unpkg.com/<package-name>");
+```
+
+Note that this does not register the custom properties for syntax and fallback values. Instead, they each have fallback values embedded into the worklet.
+
+To optionally register the custom properties, include the properties.js file as well.
+
+```html
+<script src="https://unpkg.com/<package-name>/properties.js"></script>
+```
+
+To include the CSS Paint Polyfill with unpkg:
+
+```html
+<script src="https://unpkg.com/css-paint-polyfill"></script>
+```
+
+### 2. Managing worklets via NPM
+
+Install your worklet from npm:
+
+```sh
```

Comment:
This is breaking the build because it's not a recognized language code for prism.js apparently. Try `bash`. https://prismjs.com/#supported-languages